templates/repo/release/list.tmpl

```diff
@@ -53,21 +53,25 @@
 							<div class="download">
 								<h2>{{$.i18n.Tr "repo.release.downloads"}}</h2>
 								<ul class="list">
+									{{$tagname := .TagName}}
+									{{if .Attachments}}
+										{{range $attachment := .Attachments}}
+										<li>
+											<!-- {{AppSubUrl}}/attachments/{{$attachment.UUID}} -->
```

Comment:
I will not remove this out because I want to add  for each release an option to display like github url or leave it the old way.